pennylane_lightning/src/gates/cpu_kernels/GateImplementationsPI.hpp

```diff
@@ -32,6 +32,7 @@
 #include "LinearAlgebra.hpp"
 #include "PauliGenerator.hpp"
 
+#include <bit>
```

Comment:
Not sure, just suggesting :slightly_smiling_face: 
```suggestion
```

Reply:
As above